pages/common/corepack.md

```diff
@@ -0,0 +1,36 @@
+# corepack
+
+> Zero-runtime-dependency package acting as bridge between Node projects and their package managers.
+> More information: <https://github.com/nodejs/corepack>.
+
+- Add the Corepack shims to the Node.js installation directory adding them as global commands:
+
+`corepack enable`
+
+- Add the Corepack shims to a specific directory:
+
+`corepack enable --install-directory {{path/to/directory}}`
+
+- Remove the Corepack shims from the Node.js installation directory:
+
+`corepack disable`
+
+- Prepare a specific package manager:
+
+`corepack prepare {{package}}@{{version}} --active`
+
+- Prepare the package manager configured for the project in the current path:
+
+`corepack prepare`
+
+- Use a package manager without add it as a global command:
```

Comment:
This sentense doesn't seem very complete.

Reply:
how can you explain this?
```bash
$ corepack yarn install
```
Maybe the diff of make `yarn` as a global command of the 1st example is not needed here.

Reply:
Perhaps this will read better?
```suggestion
- Use a package manager without installing it as a global command:
```